src/libproc_macro/lib.rs

```diff
@@ -334,6 +335,16 @@ impl Span {
     diagnostic_method!(help, Level::Help);
 }
 
+#[unstable(feature = "proc_macro", issue = "38356")]
+impl fmt::Debug for Span {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{:?} bytes({}...{})",
```

Comment:
I believe the lo/hi interval is half-open so `{}..{}` would be more accurate. Like in #49720 the `+` token is `bytes(82..83)`.